Source/UI/NewUI/PaymentForm/View/ExpiryDateView.swift

```diff
@@ -7,49 +7,55 @@ protocol ExpiryDateViewDelegate: AnyObject {
 
 public final class ExpiryDateView: UIView {
   weak var delegate: ExpiryDateViewDelegate?
-  
+
   /// 5 is the expected text count, for example "11/35".
   private let dateFormatTextCount = 5
-  
-  private(set) var style: CellTextFieldStyle?
   private let environment: Environment
+  private(set) var style: CellTextFieldStyle?
 
   private lazy var cardValidator: CardValidator = {
     CardValidator(environment: environment.checkoutEnvironment)
   }()
-  
-  private lazy var expiryDateView: InputView = {
+
+  private lazy var dateInputView: InputView = {
     let view = InputView().disabledAutoresizingIntoConstraints()
     view.delegate = self
     return view
   }()
-  
+
   init(environment: Environment) {
     self.environment = environment
     super.init(frame: .zero)
     // setup expiry DateView
-    addSubview(expiryDateView)
-    expiryDateView.setupConstraintEqualTo(view: self)
+    addSubview(dateInputView)
+    dateInputView.setupConstraintEqualTo(view: self)
   }
-  
+
   required init?(coder: NSCoder) {
     fatalError("init(coder:) has not been implemented")
   }
-  
+
   func update(style: CellTextFieldStyle?) {
     self.style = style
-    expiryDateView.update(style: style)
+    dateInputView.update(style: style)
   }
-  
-  private func updateErrorView(isHidden: Bool, text: String?){
-    style?.error?.isHidden = isHidden
-    style?.textfield.text = text ?? ""
-    expiryDateView.update(style: style)
+
+  private func updateErrorView(isHidden: Bool, text: String?, error: ValidationError.ExpiryDate? = nil){
```

Comment:
This method was previously deleted and replaced by `updateErrorViewStyle`. I think this has been lost someway